torch/nn/utils/parametrize.py

```diff
@@ -389,6 +394,9 @@ def right_inverse(self, X: Tensor) -> Union[Tensor, Sequence[Tensor]]
         tensor_name (str): name of the parameter or buffer on which to register
             the parametrization
         parametrization (nn.Module): the parametrization to register
+    Keyword args:
+        unsafe (bool): a boolean flag that denotes whether the parametrization breaks the invariants. Default: `False`
+            Warning: the current Module might not work anymore, enable this flag at your own risk.
```

Comment:
I think that the biggest problem of activating this flag is that the errors that the parametrization may have are not detected early and the errors messages that one gets are notably more cryptic. So perhaps write
> Warning: the parametrization is not checked for consistency upon registration, enable this flag at your own risk

Reply:
I just realised that we also need to update the `.. note:` in L377-378 reflecting this. A possible rewrite could be:
> If ``unsafe=False`` (default) both the `forward` and `right_inverse` methods will be called once to perform a number of consistency checks.
If ``unsafe=True``, then `right_inverse` will be called if the tensor is not parametrized, and nothing will be called otherwise.

Reply:
@kazhou you marked these as resolved but perhaps forgot to push the changes?

Reply:
it should be synced from phabricator to this PR? it was re-exported about 2 hrs ago